From 571e63d650c199dea27f61a0b8b1550af88b71d9 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:05:03 -0400
Subject: [usb] serial: clean up ioctl debugging

Message-id: <1410887463-20674-232-git-send-email-dzickus@redhat.com>
Patchwork-id: 95415
O-Subject: [RHEL7 PATCH 231/591] USB: serial: clean up ioctl debugging
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit 4d5147ec90531d11e7677e2c38941fc18e160641
Author: Johan Hovold <jhovold@gmail.com>
Date:   Sun Dec 29 19:22:55 2013 +0100

    USB: serial: clean up ioctl debugging

    Remove redundant ioctl debugging from subdrivers. The ioctl request code
    has already been logged by usb-serial core.

    Signed-off-by: Johan Hovold <jhovold@gmail.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/serial/f81232.c b/drivers/usb/serial/f81232.c
index 639a18f..aeb2edc 100644
--- a/drivers/usb/serial/f81232.c
+++ b/drivers/usb/serial/f81232.c
@@ -287,8 +287,6 @@ static int f81232_ioctl(struct tty_struct *tty,
  struct serial_struct ser;
  struct usb_serial_port *port = tty->driver_data;
 
- dev_dbg(&port->dev, "%s cmd = 0x%04x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   memset(&ser, 0, sizeof ser);
@@ -302,8 +300,6 @@ static int f81232_ioctl(struct tty_struct *tty,
 
   return 0;
  default:
-  dev_dbg(&port->dev, "%s not supported = 0x%04x\n",
-   __func__, cmd);
   break;
  }
  return -ENOIOCTLCMD;
diff --git a/drivers/usb/serial/ftdi_sio.c b/drivers/usb/serial/ftdi_sio.c
index 2e5cf73..b3f712f 100644
--- a/drivers/usb/serial/ftdi_sio.c
+++ b/drivers/usb/serial/ftdi_sio.c
@@ -2383,8 +2383,6 @@ static int ftdi_ioctl(struct tty_struct *tty,
 {
  struct usb_serial_port *port = tty->driver_data;
 
- dev_dbg(&port->dev, "%s cmd 0x%04x\n", __func__, cmd);
-
  /* Based on code from acm.c and others */
  switch (cmd) {
 
@@ -2401,11 +2399,7 @@ static int ftdi_ioctl(struct tty_struct *tty,
  default:
   break;
  }
- /* This is not necessarily an error - turns out the higher layers
-  * will do some ioctls themselves (see comment above)
-  */
- dev_dbg(&port->dev, "%s arg not supported - it was 0x%04x - check /usr/include/asm/ioctls.h\n",
-  __func__, cmd);
+
  return -ENOIOCTLCMD;
 }
 
diff --git a/drivers/usb/serial/io_edgeport.c b/drivers/usb/serial/io_edgeport.c
index c91481d..9c79fb2 100644
--- a/drivers/usb/serial/io_edgeport.c
+++ b/drivers/usb/serial/io_edgeport.c
@@ -1593,8 +1593,6 @@ static int edge_ioctl(struct tty_struct *tty,
  DEFINE_WAIT(wait);
  struct edgeport_port *edge_port = usb_get_serial_port_data(port);
 
- dev_dbg(&port->dev, "%s - cmd = 0x%x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCSERGETLSR:
   dev_dbg(&port->dev, "%s TIOCSERGETLSR\n", __func__);
diff --git a/drivers/usb/serial/io_ti.c b/drivers/usb/serial/io_ti.c
index b7187bf..2b3b7bf 100644
--- a/drivers/usb/serial/io_ti.c
+++ b/drivers/usb/serial/io_ti.c
@@ -2362,8 +2362,6 @@ static int edge_ioctl(struct tty_struct *tty,
  struct usb_serial_port *port = tty->driver_data;
  struct edgeport_port *edge_port = usb_get_serial_port_data(port);
 
- dev_dbg(&port->dev, "%s - cmd = 0x%x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   dev_dbg(&port->dev, "%s - TIOCGSERIAL\n", __func__);
diff --git a/drivers/usb/serial/mos7720.c b/drivers/usb/serial/mos7720.c
index 84657e0..2fe89c5 100644
--- a/drivers/usb/serial/mos7720.c
+++ b/drivers/usb/serial/mos7720.c
@@ -1885,8 +1885,6 @@ static int mos7720_ioctl(struct tty_struct *tty,
  if (mos7720_port == NULL)
   return -ENODEV;
 
- dev_dbg(&port->dev, "%s - cmd = 0x%x", __func__, cmd);
-
  switch (cmd) {
  case TIOCSERGETLSR:
   dev_dbg(&port->dev, "%s TIOCSERGETLSR\n", __func__);
diff --git a/drivers/usb/serial/mos7840.c b/drivers/usb/serial/mos7840.c
index a69da83..bc176ae 100644
--- a/drivers/usb/serial/mos7840.c
+++ b/drivers/usb/serial/mos7840.c
@@ -2070,8 +2070,6 @@ static int mos7840_ioctl(struct tty_struct *tty,
  if (mos7840_port == NULL)
   return -1;
 
- dev_dbg(&port->dev, "%s - cmd = 0x%x\n", __func__, cmd);
-
  switch (cmd) {
   /* return number of bytes available */
 
diff --git a/drivers/usb/serial/opticon.c b/drivers/usb/serial/opticon.c
index cbe779f..1b4f1d2 100644
--- a/drivers/usb/serial/opticon.c
+++ b/drivers/usb/serial/opticon.c
@@ -367,8 +367,6 @@ static int opticon_ioctl(struct tty_struct *tty,
 {
  struct usb_serial_port *port = tty->driver_data;
 
- dev_dbg(&port->dev, "%s - cmd = 0x%x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   return get_serial_info(port,
diff --git a/drivers/usb/serial/pl2303.c b/drivers/usb/serial/pl2303.c
index bedf8e4..75658b0 100644
--- a/drivers/usb/serial/pl2303.c
+++ b/drivers/usb/serial/pl2303.c
@@ -795,8 +795,6 @@ static int pl2303_ioctl(struct tty_struct *tty,
  struct serial_struct ser;
  struct usb_serial_port *port = tty->driver_data;
 
- dev_dbg(&port->dev, "%s cmd = 0x%04x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   memset(&ser, 0, sizeof ser);
@@ -810,7 +808,6 @@ static int pl2303_ioctl(struct tty_struct *tty,
 
   return 0;
  default:
-  dev_dbg(&port->dev, "%s not supported = 0x%04x\n", __func__, cmd);
   break;
  }
  return -ENOIOCTLCMD;
diff --git a/drivers/usb/serial/ssu100.c b/drivers/usb/serial/ssu100.c
index e5750be..a6fec95 100644
--- a/drivers/usb/serial/ssu100.c
+++ b/drivers/usb/serial/ssu100.c
@@ -342,8 +342,6 @@ static int ssu100_ioctl(struct tty_struct *tty,
 {
  struct usb_serial_port *port = tty->driver_data;
 
- dev_dbg(&port->dev, "%s cmd 0x%04x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   return get_serial_info(port,
@@ -352,8 +350,6 @@ static int ssu100_ioctl(struct tty_struct *tty,
   break;
  }
 
- dev_dbg(&port->dev, "%s arg not supported\n", __func__);
-
  return -ENOIOCTLCMD;
 }
 
diff --git a/drivers/usb/serial/ti_usb_3410_5052.c b/drivers/usb/serial/ti_usb_3410_5052.c
index 8718809..4a649ed 100644
--- a/drivers/usb/serial/ti_usb_3410_5052.c
+++ b/drivers/usb/serial/ti_usb_3410_5052.c
@@ -683,8 +683,6 @@ static int ti_ioctl(struct tty_struct *tty,
  struct usb_serial_port *port = tty->driver_data;
  struct ti_port *tport = usb_get_serial_port_data(port);
 
- dev_dbg(&port->dev, "%s - cmd = 0x%04X\n", __func__, cmd);
-
  if (tport == NULL)
   return -ENODEV;
 
diff --git a/drivers/usb/serial/usb-serial.c b/drivers/usb/serial/usb-serial.c
index 6091bd5..7c9dc28 100644
--- a/drivers/usb/serial/usb-serial.c
+++ b/drivers/usb/serial/usb-serial.c
@@ -405,7 +405,7 @@ static int serial_ioctl(struct tty_struct *tty,
  struct usb_serial_port *port = tty->driver_data;
  int retval = -ENOIOCTLCMD;
 
- dev_dbg(tty->dev, "%s - cmd 0x%.4x\n", __func__, cmd);
+ dev_dbg(tty->dev, "%s - cmd 0x%04x\n", __func__, cmd);
 
  switch (cmd) {
  case TIOCMIWAIT:
diff --git a/drivers/usb/serial/whiteheat.c b/drivers/usb/serial/whiteheat.c
index 36a7740..d23290e 100644
--- a/drivers/usb/serial/whiteheat.c
+++ b/drivers/usb/serial/whiteheat.c
@@ -455,8 +455,6 @@ static int whiteheat_ioctl(struct tty_struct *tty,
  struct serial_struct serstruct;
  void __user *user_arg = (void __user *)arg;
 
- dev_dbg(&port->dev, "%s - cmd 0x%.4x\n", __func__, cmd);
-
  switch (cmd) {
  case TIOCGSERIAL:
   memset(&serstruct, 0, sizeof(serstruct));
-- 
1.7.1